net/znc/files/znc.init

```diff
@@ -12,7 +15,6 @@ DISABLED=
 
 RUNAS_USER=
 RUNAS_GROUP=
-RUNAS_SHELL=
```

Comment:
That's correct, if `su` is used with a user without a shell then it would fail. As `nobody` was being used, and had no shell, then this was necessary for how it was written.